### DISCUSSION TITLE
Fails on Cluster

### DISCUSSION THREAD

**Discussion by sevvalfindik** [idaholab/raven#2265] [General]
My Heron and Raven files run without any problems on my desktop. On a cluster, I can run the test files of heron and raven. However when it comes to my own input files, Heron runs successfully but raven fails. For the sweep run, I get this message:

(   27.24 sec) CODE MODEL               : Message         -> Ouput is in "/home/fendik_21/2024_02/run/NRHES_o/sweep/32/out~inner"
(   27.24 sec) CODE MODEL               : Message         -> **************************************************
(   27.24 sec) Job Handler              : Message         ->  Process Failed <ravenframework.Runners.SharedMemoryRunner.SharedMemoryRunner object at 0x7f526809eac0> internal returnCode -1
(   27.24 sec) SAMPLER GRID             : Warning         -> There were 32 failed runs!  Run with verbosity = debug for more details.
(   27.24 sec) SAMPLER GRID             : ERROR           -> There were failed runs; aborting RAVEN.

OSError: There were failed runs; aborting RAVEN.

And for the optimization mode, I get the error below: 

OSError: The number of times class = Optimizers type= ['SPSA', 'FiniteDifference', 'ConjugateGradient', 'SimulatedAnnealing', 'GeneticAlgorithm'] is used as Optimizer is improper. Number of allowable times is 1.Got 0

In the both modes, I receive OSError. How can I solve these problems?

**Comment by wangcj05** [comment]
@sevvalfindik Could you share your input files and attach the whole screen outputs? I think @dylanjm and @PaulTalbot-INL will have more insight into these errors.

**Comment by PaulTalbot-INL** [comment]
Hi @sevvalfindik , it looks like you're getting failed `inner` runs. If you look at the note that mentions where the output is,
`( 27.24 sec) CODE MODEL : Message -> Ouput is in "/home/fendik_21/2024_02/run/NRHES_o/sweep/32/out~inner"`,
this provides the path to check for errors,
`/home/fendik_21/2024_02/run/NRHES_o/sweep/32/out~inner`.

If you look at this file, it should give you messages about why it is failing; the most common reason for failure in the `inner` loop is `infeasible solve`, which means the system has been set up in a way that there's no solution possible. Regardless, if you check this `out~inner` file, it will let you know what is failing.

If the run setup is identical between your local and HPC, we would need to know more about what is in that inner log file to know what cluster-related issues you're seeing.
